CHANGELOG

```diff
@@ -1,10 +1,18 @@
 devel
 -----
 
-* Limited module resolution in arangosh to the path from which arangosh is invoked.
-
-* Fixed BTS-621 Fixed rare case of segfault in cluster during database recovery if DBServer is
-  in upgrade mode in the same time.
+* Improved performance of inner joins with dynamic lookup conditions being
+  injected from an outer loop, for indexes of type "persistent", "hash" and
+  "skiplist". Performance improvements can be expected if the inner join is
+  invoked a lot of times with many different values fed in by the outer loop.
+  The performance improvements are due to some improved handling of index
+  lookup conditions in the internals of the VelocyPack-based index.
+
+* Limited module resolution in arangosh to the path from which arangosh is 
+  invoked.
+
+* Fixed BTS-621 Fixed rare case of segfault in cluster during database recovery
+  if DBServer is in upgrade mode in the same time.
```

Comment:
These seem to be non-related Comments right?

Don't bother if this is some kind of merge conflict right now, we should just make sure we do not get something into the CHANGELOG that actually is not merged.

Reply:
It was an intentional change, because the 80 chars line length limit was not followed in these entries, but I removed this change from my PR for improved clarity.